lib/IRGen/GenBuiltin.cpp

```diff
@@ -124,6 +124,9 @@ void irgen::emitBuiltinCall(IRGenFunction &IGF, const BuiltinInfo &Builtin,
                             ArrayRef<SILType> argTypes,
                             Explosion &args, Explosion &out,
                             SubstitutionMap substitutions) {
+  assert(Builtin.ID != BuiltinValueKind::IsOnStack &&
+         "Builtin should not exist at IRGen time");
```

Comment:
Can you add to the assert string that the builtin should be lowered in the StackSizeLimitTransform

Reply:
@eeckstein even better, I am going to move it into a helper function. I am sure we may want more builtins like this in the future, so would be good to not just throw it in an adhoc way.